packages/cache-manager/cache-type.ts

```diff
@@ -0,0 +1,6 @@
+import { CacheManagerOptions } from "./cache-manager-options";
+
+export interface ICache {
```

Comment:
I think it is preferable to add a property to the interface that will help limit the size of the cache (either by number of keys allowed or by an actual memory size). as I see it now, it seems like this implementation of the cache (after seeing both the redis and lean implementations) can potentially hold an infinite amount of responses, based on the ttl and the amount of requests in that time.

Reply:
In most cases it will hold 1-2MB of data.
It's not that much, and if someone needs a better cacher - he should use redis.
Also, the internal memory gets wiped when you restart the process.